airgun/entities/task.py

```diff
@@ -20,6 +20,22 @@ def read(self, entity_name, widget_names=None):
         view = self.navigate_to(self, 'Details', entity_name=entity_name)
         return view.read(widget_names=widget_names)
 
+    def set_chart_filter(self, char_name, index=None):
```

Comment:
Both of these new methods are missing docstrings. Please add it, along with description of expected parameters (especially for `index` here, which seems to be dictionary with specific items to navigate the table).